src/app/(home)/@section3/page.tsx

```diff
@@ -0,0 +1,93 @@
+import { Stack, Typography } from "@mui/joy";
```

Comment:
This is the fun part. It merges the old API call and component in one file. Allows for a more streamlined code. Missing error boundary.